controllers/openstackmachine_controller.go

```diff
@@ -335,20 +335,23 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, logger
 		return ctrl.Result{}, nil
 	}
 
-	if openStackMachine.Spec.FloatingIP != "" {
-		err = r.reconcileFloatingIP(computeService, networkingService, instance, openStackMachine, openStackCluster)
-		if err != nil {
-			handleUpdateMachineError(logger, openStackMachine, errors.Errorf("FloatingIP cannot be reconciled: %v", err))
-			return ctrl.Result{}, nil
-		}
-	}
-
 	if openStackCluster.Spec.ManagedAPIServerLoadBalancer {
```

Comment:
So we're dropping support for .FloatingIP on the machine spec, right?

Reply:
Yes, you are right.